website/docs/tutorials/explicit-vs-generated-ids.md

```diff
@@ -38,15 +38,15 @@ Lingui example:
 
 ### Benefits of Generated IDs
 
-1. **Avoiding "Naming Things" problem:** You don't need to come up with a name for each single phrase in the app. The natural language is used to create an ID.
+1. **Avoiding the "Naming Things" problem:** You don't need to come up with a name for each single phrase in the app.
 2. **Better Developer Experience:** a more streamlined development process with less code to write.
 3. **Preventing ID collisions:** As your application scales, explicit IDs can potentially lead to conflicts. Lingui's generated IDs ensure you steer clear of such collisions.
 4. **Avoiding Duplicates:** Duplicate messages are merged together automatically. Your translators will not have to translate the same phrases again and again. This could lead to cost savings, especially if translators charge by word count.
 5. **Smaller bundle:** The Lingui generates short ID such as `uxV9Xq` which are typically shorter than manually crafted IDs like `index.header.title`. This results in a smaller bundle size, optimizing your application's performance.
 
 ### Benefits of Explicit IDs
 
-1. **Control:** Developers have full control over the naming and assignment of explicit IDs. This control allows for precise targeting and easy maintenance of internationalization keys.
+1. **Control:** Developers have full control over the naming and assignment of explicit IDs. This allows for precise targeting and easy maintenance of internationalization keys. If your team uses a translation management system (TMS), even non-technical people can change the strings without having to touch the code.
```

Comment:
This phrase may confuse users. You probably meant here that messages in sources language (say in English) could be changed in TMS without touching sources. But that could be read and understood as only with explicit ids we have ability to change translations without developers effort.